nucypher/policy/conditions/evm.py

```diff
@@ -32,24 +32,18 @@
 from nucypher.policy.conditions.context import get_context_value, is_context_variable
 from nucypher.policy.conditions.lingo import ReturnValueTest
 
-# TODO: Move this method to a util function
-__CHAINS = {
-    60: 'ethereum',  # TODO: make a few edits for now
-    131277322940537: 'testerchain',  # TODO: this one can be moved to a pytest fixture / setup logic
-    **PUBLIC_CHAINS,
-}
-
 
-def _resolve_chain(chain: Union[str, int]) -> Tuple[str, int]:
-    """Returns the name *and* chain ID given only a name *or* chain ID"""
-    for pair in __CHAINS.items():
-        if chain in pair:
-            chain_id, chain_name = pair
-            return chain_name, chain_id
-    else:
-        raise ReencryptionCondition.InvalidCondition(
-            f"{chain} is not a known blockchain."
-        )
+# Permitted blockchains for condition evaluation
+_CONDITION_CHAINS = {
+    1: "Ethereum",  # TODO: 60 ?
+    5: "Goerli",
+    # 6: "Kotti",
+    # 42: "Kovan",
+    # 77: "Sokol",
+    # 100: "xDai",
+    137: "Polygon/Mainnet",
+    80001: "Polygon/Mumbai"
+}
```

Comment:
Can we use the names as defined by https://chainid.network/chains_mini.json in order to maintain consistency?

Reply:
Decided to remove the names altogether for this PR.